local_postgres_docker_setup.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
```

Comment:
Yes, this is mostly for development (since we would use RDS or equivalent in our lab production instance), but we were also talking about the case where collaborators and other researchers would want to spin up their own small instances of the adage-server, maybe on their machines. This case would not be a development instance, but also we cannot assume that RDS/equivalent would be available.